sync/class.jetpack-sync-module-users.php

```diff
@@ -75,6 +76,15 @@ public function sanitize_user( $user ) {
 	public function add_to_user( $user ) {
 		$user->allowed_mime_types = get_allowed_mime_types( $user );
 
+		if ( ! function_exists( 'get_user_locale' ) ) {
+			return $user;
+		}
+
+		// Only set the user local if it is different from the site local
```

Comment:
It might be a bit easier to write this as:

```php
public function add_to_user( $user ) {
	$user->allowed_mime_types = get_allowed_mime_types( $user );

	if ( ! function_exists( 'get_user_locale' ) ) {
		return $user;
	}

	// Only set the user local if it is different from the site local
	if ( get_locale() !== get_user_locale( $user->ID ) ) {
		$user->locale = get_user_locale( $user->ID );
	}
	
	return $user;
}
```

In case we need to add more meta in the future.